doorstop/core/tests/test_fixtures/001-item-references-utf8-keyword/REQ-UTF8.yml

```diff
@@ -0,0 +1,15 @@
+active: true
+derived: false
+header: ''
+level: 1.5
+links:
+- REQ001: 35ed54323e3054c33ae5545fffdbbbf5
+normative: true
+ref: ''
+references:
+- keyword: franÃ§ais
```

Comment:
it looks like you checked in an ASCII file instead of a UTF8 file? it says `franÃ§ais` but I think it makes more sense to test with `français`, right?

Reply:
You are right! I was also wondering about that but simply copied your example 😄